ansible/es-playbook.yml

```diff
@@ -5,13 +5,13 @@
   hosts: localhost
   sudo: true
   vars:
-    - ES_VER: "1.7.1"
-    - ES_SHORT_VER: "1.7"
-    - ES_MAPPER_ATTACHMENTS_VER: "2.7.0"
-    - ES_TRANSPORT_MEMCACHED_VER: "2.7.0"
-    - ES_TRANSPORT_THRIFT_VER: "2.7.0"
-    - ES_GEOCLUSTER_FACET_VER: "0.0.12"
-    - ES_IMAGE_PLUGIN_VER: "1.7.1"
+    - ES_VER: "2.0.0~beta1"
+    - ES_SHORT_VER: "2.0"
+    # - ES_MAPPER_ATTACHMENTS_VER: "3.0.0"
```

Comment:
There are some breaking changes in plugin api and seems like only mapper attachment is ready for 2.0. I will revert it later.

Sorry I'm quite slow here :)